[DNM] Add  fix your connection function

Pix1234 opened the pull request.

Commit:
+ fyc function

Commit:
Fix code style errors

RhinosF1 commented on sopel_channelmgnt/channelmgnt/__init__.py:
```diff
@@ -413,6 +413,54 @@ def invite_user(bot, trigger):
+    """Kick and ban a user from the channel, forwards user to specified channel until unbanned. The bot must be a channel operator for this command to work."""
+    chanops = get_chanops(str(trigger.sender), bot.memory["channelmgnt"]["jdcache"])
+    dodeop = False
+    forwardchan = "##fix_your_connection"
```
Use a config variable

Pix1234 replied:
Not sure how

RhinosF1 replied:
Look at the other defined options

RhinosF1 commented on sopel_channelmgnt/channelmgnt/__init__.py:
```diff
@@ -413,6 +413,54 @@ def invite_user(bot, trigger):
+            return
+        opt = Identifier(text[1])
+        nick = opt
+        mask = text[2] if any([s in text[2] for s in "!@*"]) else ''
```
See deepsource

RhinosF1 commented on sopel_channelmgnt/channelmgnt/__init__.py:
```diff
@@ -413,6 +413,54 @@ def invite_user(bot, trigger):
+                return
+            channel = opt
+            nick = text[2]
+            mask = text[3] if any([s in text[3] for s in "!@*"]) else ''
```
See deepsource

RhinosF1 reviewed (changes requested):
See deepsource and sonar cloud

deepsource-autofix[bot] committed:
Autofix issues in 1 file

Resolved issues in sopel_channelmgnt/channelmgnt/__init__.py via DeepSource Autofix

Commit:
Try to define config var

RhinosF1 commented on sopel_channelmgnt/channelmgnt/__init__.py:
```diff
@@ -451,7 +452,7 @@ def fyckb(bot, trigger):
             mask = nick + '!*@*'
         if trigger.account in chanops:
-            bot.write(['MODE', channel, '+b', mask + '$' + forwardchan])
+            bot.write(['MODE', channel, '+b', mask + '${}'.format(bot.settings.channelmgnt.support_channel)])
```
Don't use format. Not sure if it's enabled on this repo but the whole of core uses f strings.

Pix1234 replied:
How?
